code/modules/admin/verbs/deathsquad.dm

```diff
@@ -0,0 +1,208 @@
+//Deathsquad
+
+#define MAX_COMMANDOS 14
+
+GLOBAL_VAR_INIT(deathsquad_sent, FALSE)
+
+/client/proc/send_deathsquad()
+	var/client/proccaller = usr.client
+	if(!check_rights(R_EVENT))
+		return
+	if(SSticker.current_state == GAME_STATE_PREGAME)
+		to_chat(usr, "<span class='warning'>The round hasn't started yet!</span>")
+		return
+	if(GLOB.deathsquad_sent)
+		if(alert("A Deathsquad is already being sent, are you sure you want to send another?",, "Yes", "No") != "Yes")
+			return
+	else
+		if(alert("Do you want to send in the Deathsquad? Once enabled, this is irreversible.",, "Yes", "No") != "Yes")
+			return
+	message_admins("<span class='notice'>[key_name_admin(proccaller)] has started to spawn a DeathSquad.</span>")
+	log_admin("[key_name_admin(proccaller)] has started to spawn a DeathSquad.")
+	alert("This 'mode' will go on until everyone is dead or the station is destroyed. You may also admin-call the evac shuttle when appropriate. Spawned commandos have internals cameras which are viewable through a monitor inside the Spec. Ops. Office. The first one selected/spawned will be the team leader.")
```

Comment:
i wonder if the "admin-call the evac shuttle" part should go / be duplicated in chat.

Reply:
We dont call the shuttle.

We use the end round verb.